src/routes/+layout.svelte

```diff
@@ -61,34 +62,37 @@
 			<img class="layout__logo" src="/favicon.png" alt="Hollama logo" />
 		</a>
 
-		{#each SITEMAP as [href, text]}
+		{#each SITEMAP as href}
 			<a
 				class="layout__a"
 				class:layout__a--active={pathname.includes(href)}
-				class:layout__a--notification={text === 'settings' &&
+				class:layout__a--notification={href === '/settings' &&
 					$updateStatusStore.showSidebarNotification}
 				{href}
 			>
 				{#if href === '/knowledge'}
 					<Brain class="h-4 w-4" />
+					{$LL.knowledge()}
 				{:else if href === '/sessions'}
 					<MessageSquareText class="h-4 w-4" />
+					{$LL.sessions()}
 				{:else if href === '/settings'}
 					<Settings2 class="h-4 w-4" />
+					{$LL.settings()}
 				{:else if href === '/motd'}
 					<NotebookText class="h-4 w-4" />
+					{$LL.motd()}
```

Comment:
Technically there is a way to call the locales dynamically but the docs weren't Svelte specific and since we are already doing a bunch of `if` statements for the icons, might as well use that to call the individual locales.